### ISSUE TITLE
Check data for volume limits

### ISSUE THREAD

**Issue by Ross-Alexandra** [Ross-Alexandra/SENG-371-Project-2#36]
Blocked by GH-34

Before data is pulled (in a prod environment) it should first be checked for volume. That is, a check should be added once the system has navigated to the data-store that the data is not too large.

### Acceptance Criteria
 * The system can check the size of the data to be pulled from the data-store
 * If there is too much data (ie the volume is too large) then an error should be reported, and the build cancelled.
 * If there is enough data, then fetching will continue as normal.

**Comment by joshmcintosh** [comment]
Do we have hard numbers already on what is "too much" data? If not, I can create a spike ticket to find that out or add that as a requirement to #35

**Comment by Ross-Alexandra** [comment]
Feel free to create a spike, I have no idea right now (hoesntly I think an arbitrary value might be best so we dont waste time

**Comment by joshmcintosh** [comment]
Fair. I'll make a spike but it will be low priority.
